packages/api/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+# [1.0.3]
+### Feature
+* Added AssetInfo types
```

Comment:
Shall we put the changelog updates in `README.md` for now, as this is CHANGELOG from polkadot, in case of any further conflicts?

Reply:
This is not changeling from polkadot.. It was for this project.. Keeping the log in README doesn't feel right.. cause README is the first page visible on npm with changelog details.. @holygits any thought?

Reply:
Have made it in read me.. and created a ticket in backlog to update changelog and clean read me

Reply:
sounds good, should move to changelog eventually